gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -574,6 +684,39 @@ public void testGetOfflineAreas() {
         .assertValue(ImmutableList.of(TEST_OFFLINE_AREA));
   }
 
+  @Test
+  public void testStringToList() {
+    assertThat(FeatureEntity.stringToList(POLYGON_VERTICES_STRING))
+        .isEqualTo(POLYGON_VERTICES_LIST);
+  }
+
+  @Test
+  public void testStringToList_emptyValue() {
+    assertThat(FeatureEntity.stringToList("")).isEqualTo(null);
+  }
+
+  @Test
+  public void testStringToList_nullValue() {
+    assertThat(FeatureEntity.stringToList(null)).isNull();
+  }
+
+  @Test
+  public void testListToString() {
```

Comment:
`listToString` and `stringToList` are implementation details of `LocalDataStore`. Cmiiw, but I believe the tests above already exercise the code paths, so these tests are redundant.

Reply:
removed only empty list and empty string test case there now.